runelite-client/src/test/java/net/runelite/client/plugins/itemcharges/ItemChargePluginTest.java

```diff
@@ -52,6 +52,16 @@
 	private static final String PROTECT_1 = "Your dodgy necklace protects you. <col=ff0000>It has 1 charge left.</col>";
 	private static final String BREAK = "Your dodgy necklace protects you. <col=ff0000>It then crumbles to dust.</col>";
 
+	private static final String CHECK_RING_OF_FORGING_FULL = "You can smelt 140 more pieces of iron ore before a ring melts.";
+	private static final String CHECK_RING_OF_FORGING_ONE = "You can smelt one more pieces of iron ore before a ring melts.";
+	private static final String USED_RING_OF_FORGING = "You retrieve a bar of iron.";
+	private static final String BREAK_RING_OF_FORGING = "Your Ring of Forging has melted.";
+
+	private static final String CHECK_BINDING_NECKLACE_FULL = "You have 16 charges left before your Binding necklace disintegrates.";
```

Comment:
Binding necklace tests should be added in a separate commit, if not a fully separate PR.